overlay.d/05core/usr/lib/dracut/modules.d/35coreos-live/module-setup.sh

```diff
@@ -12,6 +12,11 @@ install_and_enable_unit() {
     systemctl -q --root="$initdir" add-requires "$target" "$unit" || exit 1
 }
 
+installkernel() {
+    # we do loopmounts
+    instmods loop
```

Comment:
```suggestion
    instmods -c loop
```

That way it's an explicit error if somehow dracut doesn't find it at compose time.

Reply:
done!